docs/source/projects.rst

```diff
@@ -349,103 +349,170 @@ useful if you quickly want to test a project in your existing shell environment.
+Databricks account (Community Edition is not supported) and you must have set up the
+`Databricks CLI <https://github.com/databricks/databricks-cli>`_. Find more detailed instructions
+in the Databricks docs
+(`Azure Databricks <https://docs.databricks.com/applications/mlflow/index.html>`_,
+`Databricks on AWS <https://docs.databricks.com/applications/mlflow/index.html>`_). A brief overview
+of how to use the feature is as follows:
 
 .. important::
 
-  Remote execution for MLflow projects with Docker environments is *not* currently supported.
+  Databricks execution for MLflow projects with Docker environments is *not* currently supported.
 
 Create a JSON file containing the 
 `cluster specification <https://docs.databricks.com/api/latest/jobs.html#jobsclusterspecnewcluster>`_
 for your run. Then, run your project using the command
 
 .. code-block:: bash
 
-  mlflow run <uri> -m databricks --cluster-spec <json-cluster-spec>
+  mlflow run <project_uri> -m databricks --cluster-spec <json-cluster-spec>
 
-where ``<uri>`` is a Git repository URI or a folder.
+where ``<project_uri>`` is a Git repository URI or a folder.
 
 .. _kubernetes_execution:
 
-Run a project on Kubernetes
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-MLflow projects can be executed in kubernetes clusters. Basically it uses the image created to run 
-projects in :ref:`Docker environment <project-docker-container-environments>`  and pushes it to an 
-image repository, so you need to configure MLproject with ``docker_env`` section. After that it 
-creates a Kubernetes Job that uses this published image and runs the MLflow project on kubernetes.
-A brief overview of how to configure and use this feature is as follows:
-
-In project folder you need to create a ``backend_config.json`` with the following attributes:
-
-.. code-block:: json
-
-{
-  "kube-context": "docker-for-desktop",
-  
-  "image-uri": "username/mlflow-kubernetes-example",
-
-  "kube-job-template-path": "kubernetes_job_template.yaml"
-}
-
-The ``kube-context`` attribute is the kubernetes context where mlflow will run the Job. ``image-uri`` points to the 
-registry/repository/image where the image will be pushed so kubernetes can download it and run. Remeber that mlflow 
-expects that login credentials are already stored for both kubernetes context and docker repository to push images.
-
-The ``kube-job-template-path`` points to a yaml file with the kubernetes Job/Batch specification to run the traning on 
-kubernetes. 
-See below the example available in the Docker example project. For more information about specification options please see 
-Kubernetes docs:
-`Jobs - Run to Completion <https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/>`_
-
-Have in mind that mlflow overwrites the folowing attributes in the job yaml file so it can handle jobs creation and monitor its status:
+Run a Project on Kubernetes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Run a Project -> Run an MLflow Project

Reply:
Fixed here and for Databricks (`Run an MLflow Project on Databricks`)